src/Services/ValidationService.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace ITBugTracking\Services;
+
+use ITBugTracking\Hydrators\SeverityHydrator;
+use PDO;
+
+class ValidationService
+{
+
+    public static function checkRequiredDataExists($data): bool
+    {
+        return !empty($data['reporter']) || !empty($data['title']) || !empty($data['severity']) || !empty($data['department']);
```

Comment:
These `||`s should be `&&`s, we want to check if they are all not-empty.

Reply:
done

Reply:
done